### ISSUE TITLE
Upgrade from Chromium 111.0.5563.41 to Chromium 111.0.5563.50

### ISSUE THREAD

**Issue by emerick** [brave/brave-browser#28839]
# Minor Chromium bump

https://chromium.googlesource.com/chromium/src/+log/111.0.5563.41..111.0.5563.50?pretty=fuller&n=10000

# QA tests:

* Check branding items
* Check for version bump

# Additional checks:
* No specific code changes in Brave (only line number changes in patches)

**Comment by kjozwiak** [comment]
The above requires `1.49.117` or higher for `1.49.x` verification 👍

**Comment by kjozwiak** [comment]
Quick update re: the work that remains on each platform/OS in terms of checking/verifying `C111`:

**Still requires a check re: affected `C111` areas**

* `macOS x64` - Affected areas mentioned via https://github.com/brave/brave-browser/issues/28288#issue-1570568099 need to be checked/verified
* `Linux` - Affected areas mentioned via https://github.com/brave/brave-browser/issues/28288#issue-1570568099 need to be checked/verified
* `Android Tablet` - Affected areas mentioned via https://github.com/brave/brave-browser/issues/28288#issue-1570568099 need to be checked/verified

**Affected `C111` areas completed and require a quick spot check of `111.0.5563.41`**

* `Win x64` - Just needs a simple spot check re: `111.0.5563.41` on `1.49.x` as the affected areas where checked via https://github.com/brave/brave-browser/issues/28288#issuecomment-1443641293 & https://github.com/brave/brave-browser/issues/28288#issuecomment-1444768348.
* `Android ARM` - Just needs a simple spot check re: `111.0.5563.41` on `1.49.x` as the affected areas where checked via https://github.com/brave/brave-browser/issues/28288#issuecomment-1448965363.

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.49.118 Chromium: 111.0.5563.50 (Official Build) (x86_64)
-- | --
Revision | 39f3dfc5023fe60997adc4f6cf5439cfa3e6968a-refs/branch-heads/5563@{#903}
OS | macOS Version 12.6.3 (Build 21G419)
```

**Comment by Uni-verse** [comment]
Verified on `Samsung GS 21 5G` using version(s):

```
Brave	1.49.118 Chromium: 111.0.5563.50 (Official Build) (64-bit) 
Revision	39f3dfc5023fe60997adc4f6cf5439cfa3e6968a-refs/branch-heads/5563@{#903}
OS	Android 13; Build/TP1A.220624.014
```

- [x] Version bump
- [x] Branding

<img width="500" alt="version" src="https://user-images.githubusercontent.com/17885425/223281626-2d25daa5-59fd-4b52-ad1d-f940abee7c2d.png">

**Comment by LaurenWags** [comment]
Changing to `QA/No` and `release-notes/exclude` as this is superseded by #28922

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.49.118 Chromium: 111.0.5563.50 (Official Build) (64-bit) 
-- | --
Revision | 39f3dfc5023fe60997adc4f6cf5439cfa3e6968a-refs/branch-heads/5563@{#903}
OS | Ubuntu 18.04 LTS

Verified test plan from https://github.com/brave/brave-browser/issues/28288#issue-1570568099

<details>
<summary>Verify functionality/UI for tabs and tab groups, in regular orientation and "vertical tabs" orientation_PASSED</summary>

**_regular orientation:_**

- Confirmed that user can create a new tab group via `Add tab to group` ->`New group`
![image](https://user-images.githubusercontent.com/34715963/223441092-104da08a-665b-4418-b0d7-22c35f31a5b8.png)
- Confirmed that tabs can be added to existing/already created tab groups
![image](https://user-images.githubusercontent.com/34715963/223441297-8f581cbe-bcc9-412d-819d-12433f789618.png)

- Confirmed that click on the tab group name expands/list of all the tabs added in that group

Example | Example
----------|----------
![image](https://user-images.githubusercontent.com/34715963/223441446-8c0a7925-b7e8-4075-b2bd-2b9016da999c.png)|![image](https://user-images.githubusercontent.com/34715963/223441508-a8fa190b-00f4-4601-93c7-85f825b2f0f6.png)


- Confirmed that added tabs can be removed from the tab group via `Remove from group` 
![image](https://user-images.githubusercontent.com/34715963/223441916-e214bd76-8249-4207-8883-32a8ee3732c0.png)


- Confirmed that user can ungroup the tabs via `Ungroup`
![image](https://user-images.githubusercontent.com/34715963/223444324-48d07930-7602-4ddc-b8e0-4caa378b1c00.png)

**_Vertical tabs orientation:_**

- Confirmed that vertical tabs can be enabled via the flag `#brave-vertical-tabs` 
- After enabling the flag, right click on any tab and select `Use vertical tabs` option to view vertical tabs
- Verified that sites can be added to vertical tab by clicking on the `+ New tab` 
- Right click on any site added in the vertical tab shows the tab `context menu` options
- Verified that `reading list` and `vertical tabs` cab be enabled at the same time
- Right click on any site and select `New tab to the right` context menu option and ensured new tab got added to the right
- Right click on any site and select `Add tab to reading list` context menu option and ensured site can be added to the reading list
- Right click on any site and select `Add a tab to new group` and ensured that the selected site can be added to new tab group 
- Right click on the sites in  vertical tab and select `Add a tab to new group` and then created tab (e.g `test` which I have created in my testing) group and ensured that sites got added to the tab group `test`
- Verified that created tab group can be collapsed/expanded, when the group is expanded sites added in tab groups are shown
- Right click on any site in vertical tab and select `Move tab to another window` and ensured that tab is opened in an new window
- When a tab is moved to the new window, if user try to move another tab to new window, user get option to move the tab to the existing (2nd window) or new window, ensured both options works fine
- Right click on any site in vertical tab and select `Reload` context menu option and ensured site gets reloaded/refreshed
- Right click on any site in vertical tab and select `Duplicate` context menu option and ensured duplicate tab got added to the tab group
- Right click on any site in vertical tab and select `Pin` context menu option and ensured site pinned in the tab group and right clicked on the pinned tab and ensured `Unpin` option is shown
- Ensured that pinned site can be unpinned by clicking on the `Unpin` option
- Right click on any YT site in vertical tab and select `mute` context menu option and ensured site is muted and mute option is shown in the URL bar and video is playing in an muted mode. 
- Ensured that YT videos can be unmuted by clicking on the `Unmute` option
- Ensured that `Close`, `Close other tabs`, `Close tabs to the right` and `Reopen closed tabs` works as expected
- Ensured that the tab `Bookmarls all` context menu works as expected
- Ensured that tabs can be drag and drop in an vertical tab
- Ensured that vertical tab cab be collapsed/expanded, when `vertical tabs` is expand the tabs with site name and url is shown, when vertical tab cab be collapsed, the site favicons are shown in the vertical tabs

![image](https://user-images.githubusercontent.com/34715963/223445065-7717763a-13ca-471e-a323-b0f61a525d26.png)|![image](https://user-images.githubusercontent.com/34715963/223445205-94de1946-f03b-4796-9a76-d8aa11e155b5.png)|![image](https://user-images.githubusercontent.com/34715963/223448828-6b92b4ab-e49b-4d83-8442-e4d14c7aa96f.png)|![image](https://user-images.githubusercontent.com/34715963/223451721-4eb39846-dbe9-4636-a973-5e9b96a98ddf.png)|![image](https://user-images.githubusercontent.com/34715963/223451816-1fca0b86-232c-4dc6-a580-72652fd90268.png)|![image](https://user-images.githubusercontent.com/34715963/223452059-9d2b99fe-e088-4781-a6fd-3ebd44c1eaa2.png)|![image](https://user-images.githubusercontent.com/34715963/223452152-8e42560f-e038-48e9-b8ae-1aad9590ceb1.png)|![image](https://user-images.githubusercontent.com/34715963/223715303-25540484-4332-41fe-b2eb-d6e59be42c1f.png)
--|--|--|--|--|--|--|--



</details>

<details>
<summary>brave://settings_PASSED</summary>

- Ensured that no new settings are added as part of CR 111
- Verified that click on each settings in brave://settings redirected to the intended page 
- Verified the brave://settings UI and ensured UI looks good
- Verified the icons in the brave://settings page and ensured icons are shown as expected in each settings page
- Additional testing on settings is covered in "Settings comparison" section

</details>

<details>
<summary>PDF extension_PASSED</summary>

- Ensured that PDF loaded over HTTPS at https://basicattentiontoken.org/BasicAttentionTokenWhitePaper-4.pdf
- Ensured that PDF loaded over HTTP at http://www.pdf995.com/samples/pdf.pdf
- Ensured that the PDF's can be downloaded
- Ensured taht the downloaded PDF's can be opened from system local file path

![image](https://user-images.githubusercontent.com/34715963/223715846-c927ea52-0c91-434f-95d4-285b3672767c.png)|![image](https://user-images.githubusercontent.com/34715963/223715950-47ef82ee-e0ac-4dc2-b36d-8576831ad0e2.png)|![image](https://user-images.githubusercontent.com/34715963/223716012-8de3367e-d5f8-412a-852b-341cad72212f.png)|![image](https://user-images.githubusercontent.com/34715963/223716048-05b61635-b601-427d-b3dc-2c320831c710.png)
--|--|--|--

</details>


<details>
<summary>Sidebar_PASSED</summary>

- Confirmed that `Reading list` icon is added to the sidebar menu
- Confirmed that reading list panel is shown when user click on the `Reading list` icon in the sidebar menu
- Confirmed that user can add the `list of websites` to the reading list by clicking on the `+Add current tab`
- Confirmed that user can add the `list of websites` to the reading list by right clicking on the tab and choose the option `Add tab to the reading list`
- Confirmed that `reading list panel can be hidden` by clicking on the `Reading list` icon in the sidebar menu
- Confirmed that reading list icon is shown in the URL bar, next to wallet icon
- Confirmed that user can enable/disable the reading list by clicking on the reading list icon (shown in the URL bar)
- Confirmed that reading list is shown in all the window types (Normal, PT, TOR and Guest window)
- Websites can't be added to reading list in Guest, encountered https://github.com/brave/brave-browser/issues/24930
- Confirmed that sites can be segregated to `Read` and `Unread` list
- Confirmed that sites can be deleted from the reading list

![image](https://user-images.githubusercontent.com/34715963/223716573-c986f661-0f31-4a21-9f7b-e7f29958ee14.png)|![image](https://user-images.githubusercontent.com/34715963/223716776-79765c17-5578-4d05-81f9-812fa0234112.png)|![image](https://user-images.githubusercontent.com/34715963/223717145-cf2b1050-b3e1-4c23-bc11-91fc2c37249e.png)
--|--|--

</details>

<details>
<summary>Ephemeral storage and cookie/content settings in general_PASSED</summary>

**_Ephemeral storage_**

<details>
<summary>Ephemeral storage - cross site cookies blocked, ephemeral storage enabled _PASSED</summary>

Step 1: Encountered #26817
- Test results are not same as the test page results, encountered https://github.com/brave/brave-browser/issues/26817, looks like this is expected as per https://github.com/brave/brave-browser/issues/26817#issuecomment-1319332581
![image](https://user-images.githubusercontent.com/34715963/223718304-5e5ad6f9-0b34-4ef6-9488-e715bf4ee54e.png)

Step 2: works as expected
![image](https://user-images.githubusercontent.com/34715963/223718497-b734001a-44be-443e-a341-edd35622dace.png)

Step 3: works as expected
![image](https://user-images.githubusercontent.com/34715963/223718947-6d1d5a0b-2209-4b5b-a753-5d6c1d27dd42.png)


Step 4: works as expected
![image](https://user-images.githubusercontent.com/34715963/223720702-23ca06ee-7d17-4875-9635-f536947da1af.png)


Step 5: works as expected
![image](https://user-images.githubusercontent.com/34715963/223719626-00604613-3955-4087-b69d-073bf91f52a9.png)


Step 6: works as expected
![image](https://user-images.githubusercontent.com/34715963/223719875-f0d3c4bd-4e40-4931-ad36-5fc849342cc9.png)


</details>


<details>
<summary>Ephemeral storage - All cookies blocked, ephemeral storage enabled _PASSED</summary>

Step 1: 
![image](https://user-images.githubusercontent.com/34715963/223721673-5c8cc4d1-af52-405a-8410-42067c7e3350.png)


Step 2: works as expected
![image](https://user-images.githubusercontent.com/34715963/223721825-944d45e1-26ee-4f15-9fc3-1cb04aff4ba3.png)

Step 3: works as expected
![image](https://user-images.githubusercontent.com/34715963/223721944-cf06a404-7834-440e-9745-5803bdd48235.png)


Step 4: works as expected
![image](https://user-images.githubusercontent.com/34715963/223722468-a2ddabe3-37e5-4792-adec-0928c40b93d0.png)


Step 5: works as expected
![image](https://user-images.githubusercontent.com/34715963/223722575-65f02495-89a0-4f2c-95dd-2e271b7c9f13.png)


Step 6: works as expected
![image](https://user-images.githubusercontent.com/34715963/223722723-a0baa22d-9fe5-4105-88cc-3782890756d7.png)


</details>

**_Content settings:_**

<details>
<summary>Content settings_PASSED</summary>

- Confirmed that `Shields settings` is been added in brave://settings/content (more info: https://github.com/brave/brave-browser/issues/12782)
- Added few sites to shields up and shields down list and ensured sites got added to shield settings list
- Open the sites (which are added in shield down list) in PT/TOR tab windows and ensured shields are down as expected
- Confirmed content settings are working as expected
- Confirmed cookies settings are working as expected

**_1. Block cookies content settings_**

- Open a site cnn.com
- Open content settings for cnn.com
- Block content settings for `Location` and `Window management`
- Reload cnn.com
- Ensured that `Location` and `Window management` permission are blocked for cnn.com
- Open brave://settings/cookies and `Block all cookies` 
- Reload cnn.com and ensured that all cookies are blocked on cnn.com

![image](https://user-images.githubusercontent.com/34715963/223723201-c0c0005a-55b8-4fc9-8cf0-e10cfede6460.png)|![image](https://user-images.githubusercontent.com/34715963/223723294-6d5fd16d-60a7-4a9a-8fbe-563fe649d891.png)|![image](https://user-images.githubusercontent.com/34715963/223723358-26320843-0ae8-418b-9430-546e0c5455cc.png)
--|--|--

**_2. Block scripts content settings_**

- Block the java scripts for a site via brave://settings/content/javascript
- Open the site ensured Java scripts blocked for the site
- Open a bunch of websited and ensured java script is not blocked

![image](https://user-images.githubusercontent.com/34715963/223724365-b3132d59-bf88-4d4c-bca9-ea591c96b791.png)|![image](https://user-images.githubusercontent.com/34715963/223724407-9e129972-df5b-4576-a788-59212bd791af.png)|![image](https://user-images.githubusercontent.com/34715963/223724473-1fd804ca-4540-460e-bc6c-15c8a7ca8a07.png)|![image](https://user-images.githubusercontent.com/34715963/223724519-d7751a38-97dd-473c-9299-00d9719f8a94.png)
--|--|--|--


**_3. Block autoplay global settings_**

- Block the youtube.com autoplay content global settings via brave://settings/content/autoplay
- Open youtube.com and ensured no videos are autoplayed 
- Click on a video and play the video manully, wait till the 1st video ends
- Ensured 2nd video is not autoplayed 

![image](https://user-images.githubusercontent.com/34715963/223725112-5cd0769e-4acd-4f00-84dc-369c32c03b53.png)|![image](https://user-images.githubusercontent.com/34715963/223725342-e5ab4c56-5411-4517-94cf-9a4292bff569.png)
--|--

</details>
</details>

<details>
<summary>Verify that Brave policies (such as TorDisabled) work correctly when set_PASSED</summary>

Used https://github.com/brave/brave-browser/issues/25151

![image](https://user-images.githubusercontent.com/34715963/223726520-6efe272d-54dd-4aa8-b47f-74dda13cd816.png)|![image](https://user-images.githubusercontent.com/34715963/223726685-729c32d9-ad40-4a63-94fe-c1baae063580.png)|![image](https://user-images.githubusercontent.com/34715963/223726795-bec90281-c9ad-40d4-8247-0f9ff6536993.png)|![image](https://user-images.githubusercontent.com/34715963/223726853-70d23e6a-a75c-4bf9-ab60-5c5da8251ca4.png)
--|--|--|--

</details>

**Comment by stephendonner** [comment]
## Verification `IN-PROGRESS` using `1.49.120`

**_Content settings:_**

<details>
<summary>Content settings - PENDING</summary>

- Confirmed that `Shields settings` is been added in brave://settings/content (more info: https://github.com/brave/brave-browser/issues/12782)
- Added few sites to shields up and shields down list and ensured sites got added to shield settings list
- Open the sites (which are added in shield down list) in PT/TOR tab windows and ensured shields are down as expected
- Confirmed content settings are working as expected
- Confirmed cookies settings are working as expected

**_1. Block cookies content settings_**

- Open a site cnn.com
- Open content settings for cnn.com
- Block content settings for `Location` and `Window management`
- Reload cnn.com
- Ensured that `Location` and `Window management` permission are blocked for cnn.com
- Open brave://settings/cookies and `Block all cookies` 
- Reload cnn.com and ensured that all cookies are blocked on cnn.com

Example | Example | Example | Example | Example | Example
-- | -- | -- | -- | --|---


**_2. Block scripts content settings_**

- Block the java scripts for a site via brave://settings/content/javascript
- Open the site ensured Java scripts blocked for the site
- Open a bunch of websited and ensured java script is not blocked

Example | Example | Example | Example
-- | -- | -- | ---


**_3. Block autoplay global settings_**

- Block the youtube.com autoplay content global settings via brave://settings/content/autoplay
- Open youtube.com and ensured no videos are autoplayed 
- Click on a video and play the video manully, wait till the 1st video ends
- Ensured 2nd video is not autoplayed 

Example | Example 
----------|---------


</details>
</details>

<details>
<summary>Verify that Brave policies (such as TorDisabled) work correctly when set - PENDING</summary>



Example |  Example | Example | Example | Example
----------|-----------|------------|----------|----------
